tests/e2e/runner/__tests__/minter-suite-v2/i-filtered-shared-merkle.ts

```diff
@@ -0,0 +1,81 @@
+import { describe, test, expect } from "@jest/globals";
```

Comment:
Should the filename be i-filtered-shared-merkle.test.ts

Reply:
yes - updated in [51081c1](https://github.com/ArtBlocks/artblocks-subgraph/pull/230/commits/51081c1c8e40302300691b05061e24ee8f9972e3)